### PR TITLE
Remove presentation-full-stack-tests with dependency on `components` and `testing`

### PR DESCRIPTION
**Pull request by saskliutas**
Removed presentation-full-stack-tests moved to `presentation` repository https://github.com/iTwin/presentation/pull/33. This PR will need to wait until we can use docs artifacts from `presentation` repository when building docsite because of the docs snippets extracted from moved tests.

### CODE REVIEW

File: full-stack-tests/presentation/src/Utils.ts [hunk 1]
```diff
@@ -61,3 +66,50 @@ export function getFieldsByLabel(rootFields: Field[], label: string): Field[] {
   handleFields(rootFields);
   return foundFields;
 }
+
+export async function buildTestIModel(name: string, cb: (builder: IModelBuilder) => void): Promise<IModelConnection> {
```
Comment (grigasp) [inline full-stack-tests/presentation/src/Utils.ts:70]: are these used anywhere or did you add them here for future use?

**Review by grigasp — APPROVED** [review]
LGTM

File: full-stack-tests/presentation/src/Utils.ts (hunk 1 above)
Comment (saskliutas) [reply full-stack-tests/presentation/src/Utils.ts:70]: These are used in `Hierarchies.test.ts` file only but I think they might be useful in other new tests in the future.

**Review by pmconne — APPROVED** [review]

**Comment by saskliutas** [comment]
Additionally removed `presentation-test-app` as it was moved to https://github.com/iTwin/presentation.

**Review by aruniverse — APPROVED** [review]

**Pull request merged**